const catchAsyncErrors = require("../utils/errors/asyncWrapper");

exports.findUserByEmail = catchAsyncErrors( async(req,res)=>{




    
})